<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>presidenciales</class>
 <widget class="QMainWindow" name="presidenciales">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Presidenciales</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1091</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="contenido">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>751</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#contenido{
	background-color: rgba(0,0,0,0.7);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_2{
	background-color:rgba(80,0,0,0.8);
	color: red;
	font: bold 18px;
}</string>
     </property>
     <property name="text">
      <string>CANDIDATOS PRESIDENCIALES</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="candidato1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>201</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#candidato1{
		background-color: rgba(25,0,0,0.8);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="nombre1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#nombre1{
	background-color: rgba(100, 0,0,0.8);
	color: red;
   font: bold 14px;
}</string>
      </property>
      <property name="text">
       <string>Juan Orlando Hern√°ndez</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Partido:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Votos actuales:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="votar_joh">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#votar_joh{
	color: red;
	font: bold 14px;
   background-color: rgba(200,0,0,0.2);
   border-radius: 20px;
}

QPushButton#votar_joh:hover{
	color: black;
	background-color: rgba(200,0,0,0.8);
}</string>
      </property>
      <property name="text">
       <string>VOTAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;
background-color: rgba(100,0,0,0.2);
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string>Partido Nacional</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../imagenes/joh.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_votos_joh">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#s_votos_joh{
background-color: rgba(100,0,0,0.2);
border-radius: 10px 10px 100px 100px; 
color: red;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="candidato2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>201</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#candidato2{
		background-color: rgba(25,0,0,0.8);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="nombre2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#nombre2{
	background-color: rgba(100, 0,0,0.8);
	color: red;
   font: bold 14px;
}</string>
      </property>
      <property name="text">
       <string>Salvador Nasralla</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="votar_nasralla">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#votar_nasralla{
	color: red;
	font: bold 14px;
   background-color: rgba(200,0,0,0.2);
   border-radius: 20px;
}

QPushButton#votar_nasralla:hover{
	color: black;
	background-color: rgba(200,0,0,0.8);
}</string>
      </property>
      <property name="text">
       <string>VOTAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Partido:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Votos actuales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;
background-color: rgba(100,0,0,0.2);
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string>Alianza Patriotica</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../imagenes/nasralla.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_votos_sn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#s_votos_sn{
background-color: rgba(100,0,0,0.2);
border-radius: 10px 10px 100px 100px; 
color: red;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="candidato3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>80</y>
       <width>201</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#candidato3{
		background-color: rgba(25,0,0,0.8);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="nombre3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#nombre3{
	background-color: rgba(100, 0,0,0.8);
	color: red;
   font: bold 14px;
}</string>
      </property>
      <property name="text">
       <string>Luis Zelaya Medrano</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="votar_luis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#votar_luis{
	color: red;
	font: bold 14px;
   background-color: rgba(200,0,0,0.2);
   border-radius: 20px;
}

QPushButton#votar_luis:hover{
	color: black;
	background-color: rgba(200,0,0,0.8);
}</string>
      </property>
      <property name="text">
       <string>VOTAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Partido:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Votos actuales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;
background-color: rgba(100,0,0,0.2);
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string>Partido Liberal</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../imagenes/luis.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_votos_lz">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#s_votos_lz{
background-color: rgba(100,0,0,0.2);
border-radius: 10px 10px 100px 100px; 
color: red;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="menu">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>221</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#menu{
	background-color: rgba(0,0,0,0.7);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_3{
	background-color: rgba(80,0,0,0.8);
	color: red;
	font: bold 18px;
}</string>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="regresar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>480</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#regresar{
	color: red;
	font: bold 14px;
   background-color: rgba(200,0,0,0);
   border-radius: 1px;
}

QPushButton#regresar:hover{
	color: black;
	background-color: rgba(200,0,0,0.8);
}</string>
     </property>
     <property name="text">
      <string>REGRESAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#info{
	color: red;
	font: bold 14px;
   background-color: rgba(200,0,0,0);
   border-radius: 1px;
}

QPushButton#info:hover{
	color: black;
	background-color: rgba(200,0,0,0.8);
}</string>
     </property>
     <property name="text">
      <string>INFORMACI√ìN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="info2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#info2{
	color: red;
	font: bold 14px;
   background-color: rgba(200,0,0,0);
   border-radius: 1px;
}

QPushButton#info2:hover{
	color: black;
	background-color: rgba(200,0,0,0.8);
}</string>
     </property>
     <property name="text">
      <string>SOBRE TU VOTO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="info3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#info3{
	color: red;
	font: bold 14px;
   background-color: rgba(200,0,0,0);
   border-radius: 1px;
}

QPushButton#info3:hover{
	color: black;
	background-color: rgba(200,0,0,0.8);
}</string>
     </property>
     <property name="text">
      <string>ADVERTENCIAS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_4{
	color: red;
	font: bold 16px;
}
</string>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_5{
	color: red;
	font: bold 16px;
}
</string>
     </property>
     <property name="text">
      <string>Identidad:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>460</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_6{
	color: red;
	font: bold 16px;
}
</string>
     </property>
     <property name="text">
      <string>____________________________</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_7{
	color: red;
	font: bold 16px;
}
</string>
     </property>
     <property name="text">
      <string>____________________________</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_8{
	color: red;
	font: bold 16px;
}
</string>
     </property>
     <property name="text">
      <string>____________________________</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_9{
	color: red;
	font: bold 16px;
}
</string>
     </property>
     <property name="text">
      <string>____________________________</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_10{
	color: red;
	font: bold 18px;
}
</string>
     </property>
     <property name="text">
      <string>Sesi√≥n Actual</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="s_nombre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#s_nombre{
background-color: rgba(100,0,0,0.6);
border-radius: 10px 10px 100px 100px; 
color: red;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="s_identidad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#s_identidad{
background-color: rgba(100,0,0,0.6);
border-radius: 10px 10px 100px 100px; 
color: red;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
